pkg/crd/generator/testData/config/crds/fun_v1alpha1_toy.yaml

```diff
@@ -11,6 +11,12 @@ spec:
     kind: Toy
     plural: toys
   scope: Namespaced
+  subresources:
+    scale:
+      labelSelectorPath: ""
+      specReplicasPath: .spec.replicas
+      statusReplicasPath: .status.replicas
+    status: {}
```

Comment:
Can we verify that YAML works against a real cluster :) ?

Reply:
The `labelSelectorPath ` is optional, so get it removed when value is empty string in `parsingScaleScheme()`.
Tested the CRD yaml with scale and status in minikube cluster. It is validated and installed into minikube cluster.